docset/windows/dism/add-windowscapability.md

```diff
@@ -11,7 +11,7 @@ ms.date: 12/21/2016
 ms.prod: w10
 ms.technology: powershell-windows
 ms.topic: reference
-online version: 
+online version: https://docs.microsoft.com/en-us/powershell/module/dism/add-windowscapability?view=win10-ps
```

Comment:
```suggestion
online version: https://docs.microsoft.com/powershell/module/dism/add-windowscapability?view=win10-ps
```

Reply:
You are removing /en-us/ from the URL of the document itself. But this document _is_ in English.

Reply:
Did you have chance to test that behavior? Thanks